source/AI.cpp

```diff
@@ -1505,11 +1509,12 @@ bool AI::FollowOrders(Ship &ship, Command &command) const
 		return false;
 
 	int type = it->second.type;
+	const bool isTravelOrder = (type == Orders::MOVE_TO || type == Orders::TRAVEL_TO || type == Orders::LAND_ON);
```

Comment:
not sure but wouldnt it be better named as "hasTravelOrder"?

Reply:
Probably. I'll change it.